### ISSUE TITLE
Looking for a public fundus dataset with ungradable images 

### ISSUE THREAD

**Issue by Lxinyu1999** [TheBeastCoding/glaucoma-dataset-metadata#31]
Hello, thank you for sharing the glaucoma dataset. I am a new student in the field of medical imaging analysis, and I think this is a very meaningful work.
I am looking for some public datasets that include ungradable or low quality images of the retina. I want to use these datasets for image quality assessment. Could you suggest some sources or repositories where I can find such a dataset? Thank you very much for your help.

**Comment by TheBeastCoding** [comment]
Thank you for reaching out! I personally did not work on the IQA side of
things, but there are a few great datasets for you to start with binary
classification in my public repository listing:
https://github.com/TheBeastCoding/glaucoma-dataset-metadata

Good IQA Datasets to get you started...

   - FIVES - Provides binary IQA labels on blur, low-contrast, and some
   other label I can't remember (see the original paper). Also has a
   predetermined train/test split. Very easy-to-use data and good quantity.
   - OIA-ODIR - This is a bit harder data because it is no longer binary
   labels, but text data but it does provide text notes on each fundus image,
   including if it is low quality and even if there is dust on the lens. Lots
   of data is available, but highly skewed to high-quality images.
   - EyePACS AIROGS - The largest dataset available in public space. Lots
   of amazing high-resolution images. They have a training set with glaucoma
   vs. non-glaucoma and a test set with glaucoma vs. non-glaucoma vs.
   upgradeable (1400 total images are in this class). This might have the most
   data with the upgradeable label - check and see how the authors defined
   upgradeable if you are curious: https://arxiv.org/abs/2302.01738

If you found anything helpful, please cite my work if you end up writing a
paper! https://scholar.google.com/citations?user=LaMOHJQAAAAJ&hl=en

Best,
Riley

**Comment by TheBeastCoding** [comment]
I will be adding a new feature to the readme: including the use case for each dataset for easier specialized research accessibility

**Comment by Lxinyu1999** [comment]
Thank you for your help and contribution. 😊
